### PR TITLE
259477 unit tests

### PR DESCRIPTION
**Pull request by AtosSK**

### CODE REVIEW

File: EPRN.UnitTests/Portal/Controllers/HomeControllerTests.cs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ﻿using EPRN.Portal.Controllers;
 using EPRN.Portal.Helpers.Interfaces;
 using EPRN.Portal.Services.Interfaces;
+using EPRN.Portal.ViewModels;
```
Comment (shehzad-ismail8) [inline EPRN.UnitTests/Portal/Controllers/HomeControllerTests.cs]: Are these statements required?

File: EPRN.UnitTests/Portal/Services/ExporterServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+            // Act
+            var viewModel = await _exporterHomeService.GetHomePage();
+
+            // Assert
+            Assert.IsNotNull(viewModel);
+            Assert.AreEqual(typeof(HomepageViewModel), viewModel.GetType());
+
+            Assert.IsTrue(viewModel.CardViewModels.Count() == 5);
+            Assert.IsTrue(viewModel.CardViewModels[0].Title == HomePageResources.HomePage_Waste_Title);
```
Comment (shehzad-ismail8) [inline EPRN.UnitTests/Portal/Services/ExporterServiceTests.cs]: Let's try to test for another tile just to cover a little more base and switch up the tests from the other service tests. The user role specific one is fine

**Review by shehzad-ismail8 — CHANGES_REQUESTED** [review]
Some comments

File: EPRN.UnitTests/Portal/Controllers/HomeControllerTests.cs (hunk 1 above)
Comment (AtosSK) [reply EPRN.UnitTests/Portal/Controllers/HomeControllerTests.cs]: Done.

File: EPRN.UnitTests/Portal/Services/ExporterServiceTests.cs (hunk 2 above)
Comment (AtosSK) [reply EPRN.UnitTests/Portal/Services/ExporterServiceTests.cs]: Done.

**Review by shehzad-ismail8 — APPROVED** [review]

**Pull request merged**